### ISSUE TITLE
Editable examples

### ISSUE THREAD

**Issue by anderssonjohan** [jgroth/kompendium#25]
It woulld be really nice if the code in the examples were editable in the browser. Are there plans for implementing such feature?

**Comment by jgroth** [comment]
I think it will be hard since Stencil is more of a compiler than a framework/library. However, I think I have seen some discussions in the Stencil slack channel about running the compiler in the browser, and that would probably make this easier :slightly_smiling_face:

**Comment by anderssonjohan** [comment]
@jgroth I stumbled across storybook.js in the docs of several webcomponent libraries, have you seen that? https://storybook.js.org/ 

Examples where you can play with the components using the "Knobs" tab below the canvas: 
- https://esri.github.io/calcite-app-components/?path=/story/overview-home--page 
- https://bulmil.netlify.app/

**Comment by jgroth** [comment]
Yes, I've played around with it several times but I never got it working in a satisfying way with Stencil :disappointed: Maybe it's working better these days? Last time I tried it was at the start of 2020 I think and then I gave up and started on this instead :smile:

**Comment by anderssonjohan** [comment]
@jgroth I'm currently on a slow wifi but every time I try use Storybook apps they are dead slow so I'm not for it. Although I liked the knobs stuff. That also opens up for exploratory testing of a component, even though i really like the way the "knobs" are on this for the Lime Elements button http://lime-documentation-lime-elements.s3-website-eu-west-1.amazonaws.com/versions/29.1.0/#/component/limel-button

I guess together with components like `<limel-example-value/>` you could build a utility component that can reflect over a component and bind a "form" presenting all the available "knobs" of a component? Then it's more like Storybook knobs, even though not "editable" like codepen or similar, but still really useful for playing around! :)

**Comment by jgroth** [comment]
Yeah that's a good idea. It would be quite easy to iterate all the props and generate som fields allowing a user to change them 🙂 All the metadata is available

**Comment by jgroth** [comment]
At least for primitive types, lists and objects will be harder 🤔
